### PR TITLE
Fix Kozilek's Command allowing X>0 on cascade

### LINKED ISSUE #5807: Teferi's Time Twist blink effect doesn't return card if triggered in opponent's end step
**Describe the bug**
If Teferi's Time Twist is cast during an opponent's end step, the targeted permanent does not return during your (the next) end step. Might happen with other blink-type cards/effects, but I think Teferi's Time Twist is literally the only instant that can target permanents (not just creatures) to blink, and that's what I was testing out.

**To Reproduce**
This was in a 2-player game only, not sure if it still happens with more.
1. Play a permanent (in my case this was Sands of Time, but I doubt that matters)
2. Wait until opponent's end step, and cast Teferi's Time Twist targeting Sands of Time.
3. Sands of Time is now just left in exile.

If you cast Teferi's Time Twist during your main phase (or your opponent's), the permanent comes back as expected, but obviously the point of doing it during the opponent's end phase is for it to return at yours.

The log looks like:
* Human cast Sands of Time (16)
* Sands of Time
* Human discards Plains (10)
* Turn 3 (Stella)
* Each player skips their untap step.
* Human triggered Sands of Time (16)
* At the beginning of each player's upkeep, that player simultaneously taps and untaps each tapped artifact, creature, and land they control and taps each untapped artifact, creature, and land they control. [Phase: Stella]
* Stella played Swamp (93)
* Stella didn't attack this turn.
* Human cast Teferi's Time Twist (35) targeting [Sands of Time (16)][Sands of Time (16)]
* Teferi's Time Twist (35) - Exile Sands of Time (16). Return that card to the battlefield under its owners control at the beginning of the next end step. If it enters the battlefield as a creature, it enters with an additional +1/+1 counter on it.
* Turn 4 (Human)
* Human didn't attack this turn.
* Turn 5 (Stella)

In turn 5 and the future, the targeted permanent (Sands of Time here) is still in exile.

**Expected behavior**
The targeted permanent should return on your end step.

**Smartphone (please complete the following information):**
 - Device: Google Pixel 6A
 - OS: Android 14
 - Version : 1.6.64-SNAPSHOT-08-02

### PR DESCRIPTION
**Pull request by tool4ever**
Closes #5807

### CODE REVIEW

File: forge-gui/res/cardsfolder/t/teferis_time_twist.txt [hunk 1]
```diff
@@ -2,9 +2,10 @@ Name:Teferi's Time Twist
...
-SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | Triggers$ ReturnEOT | ReplacementEffects$ EntersAsCreature | SubAbility$ DBCleanup | SpellDescription$ Return that card to the battlefield under its owner's control at the beginning of the next end step. If it enters the battlefield as a creature, it enters with an additional +1/+1 counter on it.
+SVar:DBEffect:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | RememberObjects$ RememberedLKI | SubAbility$ DBCleanup | SpellDescription$ Return that card to the battlefield under its owner's control at the beginning of the next end step. If it enters the battlefield as a creature, it enters with an additional +1/+1 counter on it.
 SVar:ReturnEOT:Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | TriggerDescription$ Return that card to the battlefield under its owner's control at the beginning of the next end step. If it enters the battlefield as a creature, it enters with an additional +1/+1 counter on it.
```
Comment (kevlahnota) [inline forge-gui/res/cardsfolder/t/teferis_time_twist.txt:6]: ReturnEOT svar still necessary?

**Review by Hanmac — APPROVED** [review]

**Comment by Agetian** [comment]
Hmm, there seems to be a conflict on Teferi's Time Twist

**Comment by Agetian** [comment]
Tried to resolve the conflict, please double check :)

**Review by Hanmac — APPROVED** [review]

**Comment by MorihiroS** [comment]
@Hanmac, @tool4ever, is this not the same bug?

koz

**Comment by Hanmac** [comment]
@MorihiroS Failed to Add to Stack is technically different, but we would need to look into this too

**Comment by MorihiroS** [comment]
@Hanmac, oh, I see. I'll have to create another issue...

**Pull request merged**